share/spice/word_map/word_map.js

```diff
@@ -11,7 +11,7 @@
             name: 'Answer',
             data: api_result,
             meta: {
-                sourceUrl: 'https://www.twinword.com',
+                sourceUrl: 'https://www.twinword.com/?mailLink=' + encodeURIComponent(api_result.encrypt_entry) + '&from=ddg',
```

Comment:
@twinword please remove the `&from=ddg` -- we can't allow you to track our users :)

You can use the referer header to know that users are coming from DuckDuckGo.com